clojure-mode.el

```diff
@@ -195,6 +195,11 @@ Out-of-the box clojure-mode understands lein, boot and gradle."
   (let ((map (make-sparse-keymap)))
     (define-key map (kbd "C-:") #'clojure-toggle-keyword-string)
     (define-key map (kbd "C-c SPC") #'clojure-align)
+    (define-key map (kbd "C-c th") #'clojure-thread)
```

Comment:
I was selfish here as I have the `cljr` prefix set to `C-c` too. Also the two letter mnemonic works so why change it?
So what are you suggesting @bbatsov `C-c C-r th`? is not that a bit tooo long?